src/morph/src/Morph.re

```diff
@@ -1,12 +1,19 @@
-let start = (~servers: list(Server.t), ~middlewares=[], handler) => {
+let start =
+    (
+      ~servers: list(Server.t('req_body, 'res_body)),
```

Comment:
Could you remind me why this is a list?

Reply:
We want to be able to run both http and https in parallell

Reply:
ah, I see. Are there any other cases? I'm just wondering if list is a good data structure. Are there any downsides to running 10 servers at the same time for instance?

Reply:
I'm not sure it makes sense to run 10 servers with the same handler and middlewares 😅

Reply:
IMO the start function should only take one server then. If someone wants to run multiple (edge case?) They always can Lwt.join them.

Reply:
You must run 2 if you want both http and https, so I don't think it's a edge case tbh

Reply:
To sum up our offline conversation - we agree that there are other possible representation of server out there and I personally think that running more than one server is an edge case indeed.

Reply:
I think I'll refactor this is a upcoming PR